sync/evmdownloader.go

```diff
@@ -244,6 +260,14 @@ func NewEVMDownloaderImplementation(
 	}
 }
 
+func (d *EVMDownloaderImplementation) GetChainID(ctx context.Context) (uint64, error) {
+	chainID, err := d.ethClient.ChainID(ctx)
+	if err != nil {
+		return 0, err
+	}
+	return chainID.Uint64(), nil
+}
```

Comment:
IMO we can directly return here without any treatment, the err can be popped up to the called and the data type too:
```suggestion
func (d *EVMDownloaderImplementation) GetChainID(ctx context.Context) (*big.Int, error) {
	return d.ethClient.ChainID(ctx)
}
```

Reply:
This is to use `uint64`  that is a more friendly type than `*big.Int`